DESCRIPTION

```diff
@@ -13,6 +13,7 @@ Encoding: UTF-8
 Depends: R (>= 3.5.0)
 Imports:
   bench,
+  dbplyr,
```

Comment:
This "shouldn't" be necessary, but in order to get it installed on CI and running it's needed. We can refactor this out, but I would prefer to defer it for now